### PR TITLE
CB-45 Create reusable component for displaying search results

### PR DESCRIPTION
**Pull request by ghost**
# JIRA TICKET NAME:
CB-45 [Create reusable component for displaying search results](url)

## SUMMARY:
Create reusable component for displaying search results by moving current search table to search history.

### CODE REVIEW

File: app/js/components/tabs/tabcomponents/conceptComponent.jsx [hunk 1]
```diff
@@ -227,34 +227,6 @@ class ConceptComponent extends Component {
...
-                                <thead>
-                                    <tr>
-                                        <th>CONCEPT AND OBSERVATIONS</th>
-                                    </tr>
-                                </thead>
-                                { concept }
-                            </table>
-                            <div>
-                             { (resultPages && resultPages.length > 1)
-                                ?   <ul className="pagination">
```
Comment (dkayiwa) [inline app/js/components/tabs/tabcomponents/conceptComponent.jsx]: Did you remove this intentionally?

File: app/js/components/tabs/tabcomponents/encounterComponent.jsx [hunk 2]
```diff
@@ -195,52 +196,6 @@ class EncounterComponent extends Component {
...
-                            </thead>
-                            <tbody>
-                            {
-                                this.state.toDisplay.map(patient => {
-                                    return (
-                                        <tr key={shortId.generate()}>
-                                            <td>{`${patient.firstname} ${patient.lastname}`}</td>
-                                            <td>{patient.age}</td>
-                                            <td>{patient.gender}</td>
-                                        </tr>);
```
Comment (dkayiwa) [inline app/js/components/tabs/tabcomponents/encounterComponent.jsx:217]: Did you remove these also intentionally?

Comment (ghost) [reply app/js/components/tabs/tabcomponents/encounterComponent.jsx:217]: Yes I did, the codes here are duplicated across components.

File: app/js/components/tabs/tabcomponents/conceptComponent.jsx (hunk 1 above)
Comment (ghost) [reply app/js/components/tabs/tabcomponents/conceptComponent.jsx]: No i think that was error on my side i will fix that now

**Pull request merged**